packages/devtools_app/test/memory/memory_screen_test.dart

```diff
@@ -104,150 +93,18 @@ void main() {
       await pumpMemoryScreen(tester);
 
       // Should be collecting live feed.
-      expect(controller.offline.value, isFalse);
+      expect(controller.offline, isFalse);
 
       // Verify Memory, Memory Source, and Memory Sources content.
       expect(find.byTooltip(ChartPaneTooltips.pauseTooltip), findsOneWidget);
       expect(find.byTooltip(ChartPaneTooltips.resumeTooltip), findsOneWidget);
 
-      expect(controller.memorySource, MemoryController.liveFeed);
-
       expect(find.text('GC'), findsOneWidget);
 
       expect(find.byType(MemoryVMChart), findsOneWidget);
 
       expect(controller.memoryTimeline.liveData.isEmpty, isTrue);
       expect(controller.memoryTimeline.offlineData.isEmpty, isTrue);
-
-      // Check memory sources available.
-      await tester.tap(find.byKey(sourcesDropdownKey));
-      await tester.pump();
-
-      // Should only be one source 'Live Feed' in the popup menu.
-      final memorySources = tester.firstWidget(
-        find.byKey(
-          sourcesKey,
-        ),
-      ) as Text;
-
-      expect(
-        memorySources.data,
-        '${controller.memorySourcePrefix}${MemoryController.liveFeed}',
-      );
-    });
-
-    testWidgetsWithWindowSize('Chart Select Hover Test', windowSize,
```

Comment:
why are we removing this test? seems we just need to switch the canned data to be loaded as live data instead of offline data, and then we can still test the chart hover card

Reply:
The test is broken. It does not test hover card. There is no card on the image.

Reply:
And there is Terry's TODO in the comments.